### PR TITLE
[*] FO : Put theme name into variable

### PR DESCRIPTION
**Pull request by martinfojtik**

### CODE REVIEW

**Comment by gskema** [comment]
Hello, thank you for your contribution. 

Could you change the variable name to  `themeName` ?

**Comment by martinfojtik** [comment]
yes, of course

File: gulpfile.js [hunk 1]
```diff
@@ -122,11 +124,11 @@ gulp.task('create-zip', function(){
         }
 
         return gulp.src([
-            './themes*/community-theme-16*/**',
+            './themes*/' + themeName + '*/**',
             './modules*/ct*/**',
             './Config.xml'
         ])
-            .pipe(zip('v' + themeVersion + '-community-theme-16.zip'))
+            .pipe(zip('v' + themeNameVersion + '-' + themeName + '.zip'))
```
Comment (gskema) [inline gulpfile.js]: `themeNameVersion` doesn't exist

Comment (martinfojtik) [inline gulpfile.js]: sorry

**Pull request merged**